repo/blob/sftp/sftp_storage.go

```diff
@@ -0,0 +1,274 @@
+package sftp
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
```

Comment:
just replace this with `crypto/rand` to resolve the issue?

Reply:
That's right. vscode autocomplete found math/rand instead of crypto/rand.